### PR TITLE
set new goal on env reset()

### PR DESCRIPTION
**Pull request by tomasruizt**

### CODE REVIEW

File: gym_roboy/envs/tests/test_robot_state.py [hunk 1]
```diff
@@ -48,3 +48,13 @@ def test_robot_normalize_max_state():
     normed_min_state = MSJ_ROBOT.normalize_state(state=MSJ_ROBOT.new_min_state())
     assert np.allclose(normed_min_state.joint_angle, -max_angles)
     assert np.allclose(normed_min_state.joint_vel, -max_vels)
+
+
+def test_normalize_between_1_and_minus1():
+    high = np.random.random()
+    low = high - np.abs(np.random.random())
+
+    high_norm = MSJ_ROBOT._normalize_between_minus1_and1(high, max_val=high, min_val=low)
```
Comment (BarisYazici) [inline gym_roboy/envs/tests/test_robot_state.py]: Isn't normalization usually in the range of 0-1?

Comment (tomasruizt) [reply gym_roboy/envs/tests/test_robot_state.py]: There is different types of normalization and scalings. This particual is between -1 and 1 for symmetry, nothing more.

**Comment by tomasruizt** [comment]
I have no opinion on this honestly. Let me know if you want it changed :)

**Comment by ALPA-Industry-and-Technology** [comment]
in ML we always normalized so that the results have been between 0 and 1

**Comment by tomasruizt** [comment]
normalization from 0 to 1 seems incompatible with our current implementation of action `scaling_factor`. We should discuss that in a new issue.

**Pull request merged**